test/test_torch.py

```diff
@@ -16124,6 +16124,9 @@ def lobpcg(*args, **kwargs):
             self.assertEqual(matmul(A, V) / E.max(), mm(matmul(B, V), (E / E.max()).diag_embed()),
                              atol=prec, rtol=0)
 
+    # we skip this test for now, because lobpcg is implemented as Function,
```

Comment:
Not sure what this is supposed to point at.. I don't think this file is 16k lines long :p

Reply:
Hmm, sorry? :) I am not sure I get it :)

Reply:
What I mean is that there is no line 16126 in `test/test_torch.py` in master :D 
And so I can't see what test you're applying this change to.
Also even extending to the next lines in this diff fails and shows nothing...

Reply:
Oh, there is a line like that. It shows 20k lines in `test/test_torch.py` for me ;)

Reply:
Apparently, the gihub web interface has a limitation on the number of lines to show.

Reply:
Haha, we should really split these files....
It's `test_lobpcg_torchscript` right? 

I am wondering if you can do something similar to what is done [here](https://github.com/pytorch/pytorch/blob/1b059f2c6de83a9c2445ab3fb21d10893f66a839/torch/autograd/grad_mode.py#L81) and use a similar guard to do:
```python
if not torch._jit_internal.is_scripting():
    return LOBPCGAutogradFunction.apply(
            A_sym, k, B_sym, X, n, iK, niter, tol, largest,
            method, tracker, ortho_iparams, ortho_fparams, ortho_bparams
        )
else:
    if A.requires_grad or B.requires_grad:
        raise RuntimeError("Script and require grads is not supported atm. If you just want to do the forward, use .detach() on A and B before calling into lobpcg")
    return _lobpcg(
            A, k, B, X,
            n, iK, niter, tol, largest, method, tracker,
            ortho_iparams, ortho_fparams, ortho_bparams
        )
```

Will that make this test pass?

Reply:
Yes, now it works! Thank you!